specification/ws/src/main/scripts/org/kaazing/specification/ws/limits/should.fail.max.lifetime.exceeded/handshake.response.and.frame.rpt

```diff
@@ -0,0 +1,31 @@
+property key ${ws:handshakeKey()}
+property validCredentials ${ws:loginBase64Encoder("joe:welcome")}
+property authHeader ${ws:append("Basic ", validCredentials)}
+
+accept http://localhost:8080/echoAuth
+accepted
+connected
+
+read method "GET"
+read version "HTTP/1.1"
+read header "Host" "localhost:8080"
+read header "Upgrade" /(?i:websocket)/
+read header "Connection" /(?i:Upgrade)/
+read header "Sec-WebSocket-Key" /(?<key>[a-zA-Z0-9+\/=]{24})/
+read header "Sec-WebSocket-Version" "13"
+read header "Authorization" ${authHeader}
+
+write status "101" "Switching Protocols"
+write version "HTTP/1.1"
+write header "Upgrade" "websocket"
+write header "Connection" "Upgrade"
+write header "Sec-WebSocket-Accept" ${ws:handshakeHash(key)}
+
+write notify HANDSHAKE_COMPLETE
+
+# clean close
+write [0x88 0x02 0x03 0xe8]
+
+write notify CLOSE_RECEIVED
```

Comment:
Again you should not notify the same barrier here. I think you can just remove this write notify instruction.